doc/services/lw_sched/index.rst

```diff
@@ -0,0 +1,195 @@
+.. _lw_sched_api:
+
+Lightweight (LW) Scheduler
+##########################
+
+.. contents::
+    :local:
+    :depth:  2
+
+The lightweight (LW) scheduler is a dedicated thread that runs at regular
```

Comment:
Could you please add a section here as to why one would use this instead of a workqueue? Coming from with no background knowledge whatsoever this is the first question that popped into my mind.

Reply:
On a related note: Please follow up on the questions from @martinjaeger regarding expanding the workqueue functionality with periodic calls.